language/linter.ts

```diff
@@ -966,7 +966,7 @@ export default class Linter {
             opcode = statement[0].value.toUpperCase();
 
             if ([
-              `ENDIF`, `ENDFOR`, `ENDDO`, `ELSE`, `ELSEIF`, `ON-ERROR`, `ENDMON`, `ENDSR`, `WHEN`, `OTHER`, `END-PROC`, `END-PI`, `END-PR`, `END-DS`, `ENDSL`
+              `ENDIF`, `ENDFOR`, `ENDDO`, `ELSE`, `ELSEIF`, `ON-ERROR`, `ON-EXCP`, `ENDMON`, `ENDSR`, `WHEN`, `WHEN-IS`, `WHEN-IN`, `OTHER`, `END-PROC`, `END-PI`, `END-PR`, `END-DS`, `END-ENUM`, `ENDSL`
```

Comment:
Should the `ON-ERROR`, `ON-EXCP` instructions not be indented inside the `MONITOR` structure?

![image](https://github.com/codefori/vscode-rpgle/assets/13275072/ef56a868-c64c-498c-9cf9-625e2a1d37e7)

Reply:
I see that IBM does not indent the `ON-ERROR` and `ON-EXCP` operation codes in the [ILE RPG Reference](https://www.ibm.com/docs/en/i/7.5?topic=codes-monitor-begin-monitor-group#zzmonit) - however, this could be dependent on the coding style, so perhaps it should be a linter option? But since this is the IBM coding style (however unlogical it is), I will accept the operations not being indented after all...

Reply:
@chrjorgensen I added two new tests in a commit which show what the linter would expect and it's similar to the IBM docs for sure. Perhaps a more granular indent configuration would be nice in the future for your suggestion.

Reply:
@worksofliam I agree - a more granular indent configuration would be nice, allowing for more code-styles.